lib/app/custom_widgets/search_list_tile.dart

```diff
@@ -17,77 +17,67 @@ class SearchListTile extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.symmetric(vertical: 8.0),
-      child: InkWell(
-        onTap: () {},
-        child: Container(
-          decoration: BoxDecoration(
-            color: AppColors.listTileGrey,
-            borderRadius: BorderRadius.circular(12),
-          ),
-          child: Row(
-            children: [
-              Expanded(
-                flex: 0,
-                child: Padding(
-                  padding: const EdgeInsets.symmetric(vertical: 24.0, horizontal: 16.0),
-                  child: Column(
-                    children: [
-                      CircleAvatar(
-                        radius: 15,
-                        child: ClipOval(
-                          child: flag.contains('.svg')
-                              ? SvgPicture.network(
-                                  flag,
-                                  fit: BoxFit.fill,
-                                  placeholderBuilder: (context) =>
-                                      const CircularProgressIndicator(color: AppColors.mainThemePink),
-                                )
-                              : CachedNetworkImage(
-                                  imageUrl: flag,
-                                  fit: BoxFit.fill,
-                                  progressIndicatorBuilder: (context, url, downloadProgress) =>
-                                      CircularProgressIndicator(
-                                          value: downloadProgress.progress, color: AppColors.mainThemePink),
-                                  errorWidget: (context, url, error) => const Icon(Icons.error, color: Colors.white),
-                                ),
-                        ),
-                      ),
-                    ],
+    return InkWell(
+      onTap: () {},
```

Comment:
Why is the `onTap` empty?

Reply:
it is left for future usage, it will redirect to details about chosen search, either team or league